duckduckgo/Android
# Allow privacy protection to be toggled on and off

Asana Issue URL: https://app.asana.com/0/414730916066338/487581092466119

## Description
Allows the user to toggle privacy protection (really just tracker blocking at this stage) on and off

## Steps to Test this PR:
TEST ONE:
1. Enter and submit a search in the omnibar
1. Open the privacy dash
1. Note that the toggle is on by default
1. Switch it off and note that the tracker icons are all still green as the detection count is zero
1. Leave the dashboard and reenter it, note the toggle is still off
1. Switch the toggle on again

TEST TWO:
1. Go to cnn.com
1. Open the privacy dash
1. Switch it off and note that the tracker icons are now red
1. Leave the dashboard and note that cnn.com is reloaded with blocking off
